act-rules/act-rules.github.io
# Update implementation documentation

The purpose of this PR is to provide clear guidance about how to produce an implementation for ACT-R. It includes

- How to use test cases
- How to submit a report
- What the report format should look like
- How is the "correctness" of an implementation determined
- How checks in an implementation are mapped to ACT-R rules

Not included:

- How to use the WCAG-EM Report tool for implementations

## Guidance for the PR (pull request) creator

**When creating PR:**
- [ ] Make sure you requesting to **pull a issue/feature/bugfix branch** (right side) to the master branch (left side).

**After creating PR:**
- [ ] Add label to indicate if it's a `Rule`, `Definition` or `Chore` (more to the administrative side)
- [ ] Add relevant project (e.g. "Q3 2018 Status") to PR
- [ ] OPTIONAL: If you want anyone in particular to review your pull request, assign them as "Reviewers".

# How to Review And Approve
- Go to the “Files changed” tab
- Here you will have the option to leave comments on different lines. 
- Once the review is completed, find the “Review changes” button in the top right, select “Approve” (if you are really confident in the rule) or "Request changes" and click “Submit review”.
